PF/PF_Session_4/Tower_Of_Hanoi_Test.java

```diff
@@ -0,0 +1,53 @@
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+
+public class Tower_Of_Hanoi_Test {
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+	}
+
+	@AfterClass
+	public static void tearDownAfterClass() throws Exception {
+	}
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	public void test() 
+	{
+		Tower_Of_Hanoi TOI=new Tower_Of_Hanoi();
+		String result=TOI.towerOfHanoi("A", "C", "B", 2);
+		String real="Move Disk 1 from A to B\nMove Disk 2 from A to C\nMove Disk 1 from B to C";
+		assertEquals("",real,result);
+	}
+	@Test
+	public void test1() 
```

Comment:
Test method name should indicate the functionality you want to test in the particular method.